### PR TITLE
handle no connection for calling retrofit request for home fragment

### PR DESCRIPTION
**Pull request by AhmedAbdallah125**

### CODE REVIEW

**Review by AbdulrahmanAli19 — APPROVED** [review]

File: app/src/main/java/com/iti/android/team1/ebuy/ui/home/view/HomeFragment.kt [hunk 1]
```diff
@@ -4,6 +4,7 @@ import android.content.ClipData
 import android.content.ClipboardManager
 import android.content.Context.CLIPBOARD_SERVICE
 import android.os.Bundle
+import android.util.Log
```
Comment (MohabElziny) [inline app/src/main/java/com/iti/android/team1/ebuy/ui/home/view/HomeFragment.kt:7]: remove Log

File: app/src/main/java/com/iti/android/team1/ebuy/NoInternetFragment.kt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+
+class NoInternetFragment : Fragment() {
+
+    private var _binding: FragmentNoInternetBinding? = null
+    private val binding get() = _binding!!
+    private val viewModel: ConnectionViewModel by viewModels()
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+    }
```
Comment (MohabElziny) [inline app/src/main/java/com/iti/android/team1/ebuy/NoInternetFragment.kt:23]: What is the purpose of this fragment?

**Review by MohabElziny — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/iti/android/team1/ebuy/ui/home/view/HomeFragment.kt (hunk 1 above)
Comment (AhmedAbdallah125) [reply app/src/main/java/com/iti/android/team1/ebuy/ui/home/view/HomeFragment.kt:7]: Done

File: app/src/main/java/com/iti/android/team1/ebuy/NoInternetFragment.kt (hunk 2 above)
Comment (AhmedAbdallah125) [reply app/src/main/java/com/iti/android/team1/ebuy/NoInternetFragment.kt:23]: Removed is done

**Review by MohabElziny — APPROVED** [review]

**Pull request merged**